rootfs/api/models/app.py

```diff
@@ -225,6 +225,18 @@ def create(self, *args, **kwargs):  # noqa
             raise ServiceUnavailable('Kubernetes resources could not be created') from e
 
         try:
+            # In order to create an ingress, we must first have a namespace.
+            if settings.EXPERIMENTAL_NATIVE_INGRESS():
```

Comment:
since `settings.EXPERIMENTAL_NATIVE_INGRESS` is a variable this will break with the trailing parentheses.